Calendar/Get-CalendarDiagnosticObjectsSummary.ps1

```diff
@@ -374,13 +433,21 @@ function BetterThanNothingCNConversion {
         $cNameMatch = $PassedCN -split "cn="
 
         # Normally a readable name is sectioned off with a "-" at the end.
-        # example /o=ExchangeLabs/ou=Exchange Administrative Group (FYDIBOHF23SPDLT)/cn=Recipients/cn=d61149258ba04404adda42f336b504ed-Delegate
+        # Example /o=ExchangeLabs/ou=Exchange Administrative Group (FYDIBOHF23SPDLT)/cn=Recipients/cn=d61149258ba04404adda42f336b504ed-Delegate
         if ($cNameMatch[-1] -match "-[\w* -.]*") {
-            Write-Verbose "BetterThanNothingCNConversion: Returning : [$($cNameMatch[-1])]"
-            return $cNameMatch.split('-')[-1]
+            Write-Verbose "BetterThanNothingCNConversion: Matched : [$($cNameMatch[-1])]"
```

Comment:
Wouldn't it be better if you just do the last X number of the CN? If we don't want it to be so long, that might be an idea.

Reply:
Unfortunately, there is enough variation in the CN formation, that the last X often is worse than this convoluted system that we have. This is likely getting moved into the Dev code in a few months, so it will be removed / clean up from here.